.github/workflows/upload.yml

```diff
@@ -75,6 +76,6 @@ jobs:
             SHORT_REFNAME="${REFNAME:0:40}"
             CHANNEL="testing-${SHORT_REFNAME//\//_}"
           fi
-          conan create -s build_type=${{ matrix.build_type }} -b missing . osp/${CHANNEL}
+          conan create -s build_type=${{ matrix.build_type }} -b missing . --user=osp --channel=${CHANNEL}
```

Comment:
The current README simply refers users to the CI workflow setup for instructions. This is good in the sense that the instructions will always be up-to-date and known to work, but it is unfortunate in the sense that it requires users to have at least some understanding of the GitHub workflow definition files.

Do I understand you correctly that you wish me to *add* the instructions to the README? (If not, I still have to update the link to the workflow file, since I changed its name.)